tests/utils/decorators/test_decorators.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Import modules
+import pytest
+import numpy as np
+from numbers import Number
+
+# Import from package
+from pyswarms import cost
+
+
+@pytest.mark.parametrize(
+    "objective_func",
+    [np.sum(), np.prod()]
+)
+def test_cost_decorator(objective_func, particles):
+    def cost_func_w_o_decorator(x):
+        n_particles = x.shape[0]
+        cost = [objective_func(x[i]) for i in range(n_particles)]
+        return cost
+
+    @cost
+    def cost_func_w_decorator(x):
+        cost = objective_func(x)
+        return cost
+
+    w_o_decorator = cost_func_w_o_decorator(particles)
+    w_decorator = cost_func_w_decorator(particles)
+
+    assert isinstance(w_decorator, Number)
```

Comment:
Hmmm, I'm not sure why we need to test if the output is a `Number`. Maybe we should test the shape of the resulting array? Ensure that it's `(n_particles, )`?